test/tool/resources/concat.json

```diff
@@ -0,0 +1,272 @@
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [4, 3, 1, 2],
+            "data": "float32Axis0"
+          },
+          "type": "float32"
+        }
+      ],
+      "axis1": [
+        {
+          "inputs": {
+            "shape": [[2, 2, 1, 2], [2, 2, 1, 2], [2, 2, 1, 2]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [2, 6, 1, 2],
+            "data": "float32Axis1Three4D"
+          },
+          "type": "float32"
+        }
+      ],
+      "axis2": [
+        {
+          "inputs": {
+            "shape": [[1, 2, 2, 1], [1, 2, 8, 1], [1, 2, 2, 1]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [1, 2, 12, 1],
+            "data": "float32Axis2Three4D"
+          },
+          "type": "float32"
+        }
+      ],
+      "axis3": [
+        {
+          "inputs": {
+            "shape": [[1, 3, 1, 1], [1, 3, 1, 1], [1, 3, 1, 2], [1, 3, 1, 4]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [1, 3, 1, 8],
+            "data": "float32Axis3Four4D"
+          },
+          "type": "float32"
+        }
+      ]
+    },
+    "5D": {
+      "axis0": [
+        {
+          "inputs": {
+            "shape": [[1, 2, 1, 1, 3], [1, 2, 1, 1, 3], [1, 2, 1, 1, 3], [1, 2, 1, 1, 3]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [4, 2, 1, 1, 3],
+            "data": "float32Axis0"
+          },
+          "type": "float32"
+        }
+      ],
+      "axis1": [
+        {
+          "inputs": {
+            "shape": [[1, 2, 3, 1, 1], [1, 6, 3, 1, 1]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [1, 8, 3, 1, 1],
+            "data": "float32Axis1Two5D"
+          },
+          "type": "float32"
+        }
+      ],
+      "axis2": [
+        {
+          "inputs": {
+            "shape": [[1, 2, 1, 1, 2], [1, 2, 2, 1, 2], [1, 2, 3, 1, 2]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [1, 2, 6, 1, 2],
+            "data": "float32Axis2Three5D"
+          },
+          "type": "float32"
+        }
+      ],
+      "axis3": [
+        {
+          "inputs": {
+            "shape": [[3, 1, 1, 1, 2], [3, 1, 1, 3, 2]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [3, 1, 1, 4, 2],
+            "data": "float32Axis3Two5D"
+          },
+          "type": "float32"
+        }
+      ],      
+      "axis4": [
+        {
+          "inputs": {
+            "shape": [[1, 2, 1, 1, 4], [1, 2, 1, 1, 8]],
+            "data": "float64Data"
+          },
+          "expected": {
+            "shape": [1, 2, 1, 1, 12],
+            "data": "float32Axis4Two5D"
+          },
+          "type": "float32"
+        }
+      ]
+    }
+  },
```

Comment:
Above `tests` part is reused the [`tests` part](https://github.com/BruceDai/wpt/blob/add_webnn_concat_tests/webnn/resources/test_data/concat.json#L2) of test_data file for WebNN `concat` tests on WPT.